src/Form.jsx

```diff
@@ -202,9 +202,9 @@ export default class Form extends React.Component {
         focused: false,
         disabled: prevDisabled
       }
-    }).then(() => {
+    }).then(({ nextProps }) => {
       /* Invoke custom onBlur handler */
-      if (onBlur) onBlur();
+      if (onBlur) onBlur(event, nextProps, this.props);
```

Comment:
Please, use the same argument structure as mentioned before:
```js
onBlur({ event, fieldProps, formProps: this.props });
```
So that any custom event handler always expects:
* Native event
* Field properties at the moment
* Form properties at the moment

I am also thinking about adding `fields` to the arguments list, but this may be a future request.